xml/cap_depl_eks.xml

```diff
@@ -36,11 +31,23 @@
 
   <para>
    When you create your cluster, use node sizes that are at least a
-   <literal>t2.large</literal>.
+   <literal>DS4_v2</literal>. The <literal>NodeVolumeSize</literal> must be a
```

Comment:
Isn't `DS4_v2` the Azure node naming scheme? This appears to be the EKS section…

Reply:
:heavy_check_mark: